api/src/main/java/com/datatorrent/api/LocalMode.java

```diff
@@ -18,34 +18,62 @@
  */
 package com.datatorrent.api;
 
-import java.util.Iterator;
-import java.util.ServiceLoader;
-
+import org.apache.apex.api.Launcher;
 import org.apache.hadoop.conf.Configuration;
 
 /**
  * Local mode execution for single application
  *
+ * @deprecated
```

Comment:
This causes deprecation warnings in EmbeddedAppLauncher. I don't think the new class should extend deprecated code.